tiled/config.py

```diff
@@ -304,7 +304,16 @@ def parse_configs(src_file: Union[str, Path]) -> Config:
             if f.is_file() and f.suffix == ".yml":
                 new_config = parse(f)
                 if common := new_config.keys() & conf.keys():
-                    raise ValueError(f"Duplicate configuration for {common} in {f}")
+                    # These specific keys can be merged from separate files.
+                    # This can be useful for config.d-style where different
+                    # files are managed by different stages of configuration
+                    # management.
+                    mergeable_lists = {"trees", "specs"}
```

Comment:
From the old config merge function (ln 247-251), it looks like `media_types`, `file_extensions` and `allow_origins` were also merged instead of being exclusive. Should they be allowed here as well? It might be awkward if they need merging in different ways, extend vs update etc

Reply:
I agree: extending this to objects (`media_types` and `file_extensions`) makes things more complex. Let's leave these out. (NSLS-II does not rely on them.) If someone yells, we can revisit and re-instate if needed.